pkg/action/cl001/ac009/job.go

```diff
@@ -0,0 +1,142 @@
+package ac009
+
+import (
+	"fmt"
+
+	batchapiv1 "k8s.io/api/batch/v1"
+	apiv1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/awscnfm/v12/pkg/project"
+)
+
+const (
+	jobPriorityClass  = "giantswarm-critical"
+	iamRoleAnnotation = "iam.amazonaws.com/role"
+)
+
+func jobNetworkPolicy() *networkingv1.NetworkPolicy {
+	name := fmt.Sprintf("%s-kiam-test", project.Name())
+
+	np := &networkingv1.NetworkPolicy{
+		TypeMeta: apismetav1.TypeMeta{
+			Kind:       "Job",
+			APIVersion: batchapiv1.GroupName,
+		},
+		ObjectMeta: apismetav1.ObjectMeta{
+			Name:      name,
+			Namespace: kubeSystemNamespace,
+			Labels: map[string]string{
+				"app":        name,
+				"created-by": project.Name(),
```

Comment:
What about using the [`managed-by`](https://github.com/giantswarm/apiextensions/blob/master/pkg/label/id.go#L22) label we officially use.

Reply:
done